scripts/anonymizeDb.ts

```diff
@@ -0,0 +1,67 @@
+import config from '../utils/config';
+import knexModule from 'knex';
+import { psychologistsTable } from '../db/tables';
+import faker from 'faker';
+
+faker.locale = 'fr';
+
+const PROD_HOSTNAME = 'https://santepsy.etudiant.gouv.fr';
+
+const knexConfig = require('../knexfile');
+
+const knex = knexModule(knexConfig);
+
+const fakeFirstnames = (real: string) : string => {
+  const nbFirstnames = real ? real.split(' ').length - 1 : 0;
```

Comment:
isn't firstNames mandatory ? 
you could use split then a reduce :)

Reply:
Yes my bad, I had an issue before because I mistyped "firstNames", I'll remove it